### ISSUE TITLE
[Code Formatter] can't format valid code

### ISSUE THREAD

**Issue by Sorien** [cnpack/cnwizards#105]
```pascal
procedure TestProc;
var
  Test: Boolean;
begin
  var Foo := not Test;
end
```

Delphi 10.3 Version: 26.0.32429.4364 
CnPack Version: 1.1.8.980 Build 2019.07.01 (can't try it on a newer version https://github.com/cnpack/cnwizards/issues/100)

**Comment by Sorien** [comment]
same form anonymous function

```pascal
    var TestProc: TAnonymousFunc :=
      procedure()
      begin
      end;
```

**Comment by shanzhashu** [comment]
Thanks. Fixed in 1.1.8.987 nightly-build version.

**Comment by Sorien** [comment]
I have to build cnpack from sources and it seems to be working, thx
